docs/reset-mfa.md

```diff
@@ -1,4 +1,17 @@
 # Reset MFA
 
-- [Reset security verification for WOG account](reset-security-verification-for-wog-account)
-- [Reset security verification for TechPass](reset-techpass-mfa-for-new-device)
+TechPass users need to reset MFA when they do any of the following:
+
+- Change their mobile phone.
+- Delete Authenticator app from their mobile phone.
+- Get a new TechPass account.
+
+
+Users whose TechPass domain is ```techpass.gov.sg```,  will only have TechPass profile(account) and must [Reset TechPass MFA](reset-techpass-mfa-for-new-device) when needed.
+
+Users whose TechPass domain is not ```techpass.gov.sg``` will have two profiles(accounts) - WOG and TechPass. Such users must do the following:
+
+- [Reset MFA for their WOG profile](reset-security-verification-for-wog-account) only if you have changed the mobile phone or deleted the Authenticator app from your mobile phone.
```

Comment:
```suggestion
- [Reset WOG MFA profile](reset-security-verification-for-wog-account) only if you have changed the mobile phone or deleted the Authenticator app from your mobile phone.
```

Reply:
agreed, but will it be better if I replace all **WOG** with **SGGovt M365** ?

Reply:
noted, thanks!